File: easy/art/art.java
package ejercicios;

import java.util.Scanner;

/**
 *06/05/2021
 * @author Andre Peralta
 */
public class metodo {
    public static void main(String[] args) { 

        Scanner scan = new Scanner(System.in);
        
        System.out.println("Number of colors : ");
        int colors = scan.nextInt();
        
        double cost = (40 + colors*5);
        
        cost += cost/10;
        
        cost = Math.ceil(cost);
        
        System.out.println((int)cost);
   }
}

File: easy/art/art.c
#include <stdio.h>
#include <math.h>

void main() {

    double cost = 40;

    int n_colors;

    printf("Introduce el número de pinturas:");
    scanf("%d",&n_colors);

    cost += n_colors*5;
    cost += cost/10;

    printf("%d",(int)ceil(cost));
}
